documentation/modules/tracing/ref-tracing-environment-variables.adoc

```diff
@@ -38,6 +38,13 @@ If you are using another tracing implementation, xref:proc-enabling-tracing-type
 
 |===
 
+WARNING: the OpenTelemetry specification doesn't define any https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md[SDK environment variables^] for enabling TLS on the Jaeger exporter used by default on the Kafka components.
+It means that you cannot enable TLS encryption on the gRPC port of your Jaeger backend, otherwise the exporter will not be able to connect.
+If you are using the Jaeger operator to deploy the Jaeger instance, set the `collector.grpc.tls.enabled: false` property under the `spec.allInOne.options` in the `Jaeger` custom resource.
```

Comment:
```suggestion
WARNING: The Kafka Bridge uses the Jaeger exporter by default. 
SDK environment variables for enabling TLS are not defined in the exporter specification. 
This means that you cannot enable TLS encryption on the gRPC port of your Jaeger endpoint. 
If you do, the exporter cannot connect. 
If you are using the Jaeger operator to deploy the Jaeger instance, set the `collector.grpc.tls.enabled` property to `false` under `spec.allInOne.options` in the `Jaeger` custom resource.
```

Reply:
Can we add the two links as additional resources at the end of the sections?

[role="_additional-resources"]
.Additional resources

* {otel-sdk-env-vars}
* {otel-oltp-en-vars}